docs/data_model.md

```diff
@@ -58,7 +58,7 @@ decentralized authorization, and how to set up role-based authorization.
 * Policy - a set of assertions that govern usage of resources
 * Assertion - a quadruple of &lt;effect, role, action, resource&gt;
 * Principal - an authenticated user or service that assumes one or more roles
-* User - a user identity that is authenticate by a particular authority like Bouncer
+* User - a user identity that is authenticate by a particular authority
```

Comment:
Saw we removed references to Bouncer a couple of months ago so I removed it from here too